staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -811,7 +811,7 @@ func describePod(pod *corev1.Pod, events *corev1.EventList) (string, error) {
 		}
 		printLabelsMultiline(w, "Labels", pod.Labels)
 		printAnnotationsMultiline(w, "Annotations", pod.Annotations)
-		if pod.DeletionTimestamp != nil {
+		if pod.DeletionTimestamp != nil && (pod.Status.Phase == corev1.PodRunning || pod.Status.Phase == corev1.PodPending) {
```

Comment:
Same, could we use `!util.IsPodPhaseTerminal` [here](https://github.com/kubernetes/kubernetes/blob/9e2075b3c87061d25759b0ad112266f03601afd8/pkg/api/v1/pod/util.go#L317)?

Reply:
`!util.IsPodPhaseTerminal` is not just `PodRunning` or `PodPending`, is that ok?

Reply:
It only differs by the Unknown which is depracated for a long time already: https://github.com/kubernetes/kubernetes/blob/c8a51aaffc94da6e7a66d8984890d3111e361a54/pkg/apis/core/types.go#L2777-L2793, so I wouldn't worry about it, and would prefer keeping the code simple.